app.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const app = express();
+
+const {
+  getTopics,
+} = require('./controllers/topics.controllers');
+
+app.use(express.json());
+
+app.get('/api/topics', getTopics);
+
+app.all('*', (request, response, next) => {
+  response.status(404).send({ msg: 'Endpoint not found' });
+})
+
+// Handle application errors.
+
```

Comment:
Although it is fine to have the 500 catch all error handling middleware function below, the PSQL error handling and custom error handling isn't technically needed yet and is extra to this ticket